src/Runner.Worker/Handlers/CompositeActionHandler.cs

```diff
@@ -70,6 +74,9 @@ public async Task RunAsync(ActionRunStage stage)
                     // Set GITHUB_ACTION_PATH
                     step.ExecutionContext.SetGitHubContext("action_path", ActionDirectory);
 
+                    // Add Expression Functions
```

Comment:
Where does each nested step display name get evaluated? Is it always JIT or do we do a first-pass at the beginning of the job?